unified-codes/apps/web/src/reducers/DetailReducer.ts

```diff
@@ -1,10 +1,10 @@
-import { DETAIL_ACTIONS, IDetailAction, IDetailUpdateEntityAction } from '../actions';
+import { DETAIL_ACTIONS, IDetailAction, IDetailUpdateEntityAction, IDetailFetchEntityAction, IDetailFetchSuccessAction, IDetailFetchFailureAction } from '../actions';
 import { IDetailState } from '../types';
 
 const initialState = (): IDetailState => {
```

Comment:
Whoops, this was me! 

Recommend changing this to an object instead of function (see `ExplorerReducer` for e.g.).

Reply:
Haha I didn't even notice. I'll update it